pandas/io/formats/style_render.py

```diff
@@ -464,6 +469,16 @@ def _generate_col_header_row(self, iter: tuple, max_cols: int, col_lengths: dict
             )
         ]
```

Comment:
Is this a breaking change? For column level names, it removes the `index_name` class and adds in the `columns_names` class in it's place. If users were purposefully doing styles with `.index_name` selectors with the intention of applying those to column level names, it will break that behavior.

Reply:
Well it will break the example in the user guide because that styles class `index_name`.

However, this should be an easy fix:

```
index_names = {
    'selector': '.index_name, .col_names',
    'props': 'font-style: italic; color: darkgrey; font-weight:normal;'
}
```

As long as this break is documented I think including the additional functionality is worth it in long run.